addons/crm/models/crm_lead.py

```diff
@@ -283,21 +283,23 @@ def _compute_company_id(self):
                 if lead.user_id and proposal not in lead.user_id.company_ids:
                     proposal = False
                 # inconsistent
-                if lead.team_id.company_id and proposal != lead.team_id.company_id:
+                elif lead.team_id.company_id and proposal != lead.team_id.company_id:
```

Comment:
Why changing this ? This is not really a if-elif heuristic, more a collection of use case that we want to avoid and that trigger the reset of the company.

Reply:
This was suggested by @flch-odoo , and does indeed make things a tiny tiny bit faster I guess? From a logic point of view, both are equivalent